ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -1599,6 +1599,15 @@ IE_CPU.deformable_convolution_2D_integral_offsets_groups_and_deforgroups
 IE_CPU.deformable_convolution_2D_real_offsets_groups_basic
 IE_CPU.deformable_convolution_2D_real_offsets_groups_complex
 IE_CPU.deformable_convolution_2D_real_offsets_groups_and_deforgroups
+IE_CPU.deformable_convolution_opset8_2D_zeroed_offsets_groups_basic
```

Comment:
Looks like after this PR we have some regressions. Am I right?

Reply:
yes, in previous implementation we fallback on reference implementation def_conv, but now we convert opset8 def_conv -> opset1 def_conv and execute version from CPU plug-in which not support ```group > 1```. As far as I now we have no real cases with ```group > 1``` so it's broke tests only. If it necessary I can add callback which will be check that ```group > 1``` and perform transformation and tests should pass.
Do we need these?

Reply:
I don't think we need it right now. Lets merge as is and enable these tests as part of PR with full Deformable_Convolution_8 support.
cc'ed @yury-intel